tests/eas/load_tracking.py

```diff
@@ -45,6 +45,15 @@ class _LoadTrackingBase(LisaTest):
         'modules': ['cpufreq', 'cgroups'],
     }
 
+    @memoized
+    @staticmethod
+    def _get_cpu_capacity(test_env, cpu):
+        if test_env.nrg_model:
+            return test_env.nrg_model.get_cpu_capacity(cpu)
+        else:
```

Comment:
I would simply add an empty line before the last `return`
```python
        if test_env.nrg_model:
            return test_env.nrg_model.get_cpu_capacity(cpu)

        return test_env.target.read_int(
            '/sys/devices/system/cpu/cpu{}/cpu_capacity'.format(cpu))
```

That reads why better and it also better marks that you are done if you enter the IF branch.

Maybe it's just mine obsession but I cannot really get an `else` after a `return` :fearful: